Stratis.Bitcoin/Base/ChainState.cs

```diff
@@ -80,5 +80,13 @@ public ChainedBlock HighestPersistedBlock
         {
             get; set;
         }
+
+        /// <summary>
+        /// Represents the last block stored to the index repository
+        /// </summary>
+        public ChainedBlock HighestIndexedBlock
```

Comment:
See HighestPersistedBlock in the BlockStoreLoop and HighestIndexedBlock in the IndexStoreLoop. It's a like-for-like change in support of deriving the index store from the block store, as we can't update the same variable in both cases. The variables are used to display the block store height and/or index store height in the console subject to whether each feature is present. If we never want to see it in the console I can remove it or maybe it can be addressed differently for both these features.